### ISSUE TITLE
Multiprotocol: How to enable feature "bind on channel"?

### ISSUE THREAD

**Issue by patman13** [FrSkyRC/ETHOS-Feedback-Community#3318]
In OpenTX there is a check box to enable "bind on channel".
In Ethos I can't find this option.
Is there a way to activate this feature using Ethos like lua script or something?

Regards,
Patman

**Comment by RealTadango** [comment]
It does not exist. MPM support in Ethos is very basic ...

**Comment by patman13** [comment]
I thought there would be a parameter I could set inside the lua Script.

Like this at top of the script:

`
local FAILSAFE = 1
local TELEMETRY = 2`

**Comment by patman13** [comment]
@bsongis Could you add a checkbox to enable "bind on channel"?
--> https://www.multi-module.org/using-the-module/binding#bind-on-channel

**Comment by bsongis-frsky** [comment]
If you have time to test the nightlies, I could add them to 1.5.0, but I want to be sure not to introduce a regression!

**Comment by bsongis-frsky** [comment]
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/45d10a7f-7424-406f-9b81-1813a3551f33)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/5464266f-fc90-46c7-8fcb-4345843ad8c9)

**Comment by bsongis-frsky** [comment]
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/46e16870-1524-476b-bdcb-3614c7cbf19b)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/ff89237e-3c33-4161-b20f-3537e872da3d)

**Comment by bsongis-frsky** [comment]
```lua
-- options sent on byte 26
local DISABLE_TELEMETRY_OPTION = {name="Disable telemetry", type="checkbox", min=0, max=0x02000000}
local DISABLE_CH_MAPPING_OPTION = {name="Disable channel mapping", type="checkbox", min=0, max=0x01000000}

-- options sent on byte 1
local BIND_ON_CHANNEL_OPTION = {name="Bind on channel", type="checkbox", min=0, max=0x400000}

-- options sent on byte 2
local LOW_POWER_OPTION = {name="Low power", type="checkbox", min=0, max=0x8000}

-- options sent on byte 3
local RF_TUNE_OPTION = {name="RF tune", min=-128, max=127}
local DSM2_ENABLE_MAX_THROW_OPTION = {name="Enable max throw", type="checkbox", min=0, max=0x80}
local DSM2_SERVO_REFRESH_RATE_OPTION = {name="Servo refresh rate", type="choice", max=0x40, values={{"22ms", 0x00}, {"11ms", 0x40}}}
local FLYSKY_SERVO_REFRESH_RATE_OPTION = {name="Servo refresh rate", min=0, max=70, display=function(value) return (value * 5 + 50) .. "Hz" end}
```

Then

```lua
system.registerMultimoduleProtocol("Assan", 24, {options={BIND_ON_CHANNEL_OPTION, DISABLE_TELEMETRY_OPTION, LOW_POWER_OPTION}})
...
system.registerMultimoduleProtocol("Spektrum", 6, {variants={"DSM2-1F", "DSM2-2F", "DSMX-1F", "DSMX-2F", "AUTO", "DSMR-1F"}, minChannels=3, maxChannels=12, options={DSM2_ENABLE_MAX_THROW_OPTION, DSM2_SERVO_REFRESH_RATE_OPTION, DISABLE_TELEMETRY_OPTION, DISABLE_CH_MAPPING_OPTION, LOW_POWER_OPTION}})
```

**Comment by patman13** [comment]
@bsongis-frsky I would test ist. In which nightly it would be included?

**Comment by bsongis-frsky** [comment]
Thanks!
We have a problem with Ethos Suite and nightlies right now, because of a last minute fix for RC
It should be OK on nightlies between Monday and Tuesday

**Comment by patman13** [comment]
@bsongis-frsky I saw you put this issue into 1.5.1 Is the function included in a nightly. Otherwise I will not flash and test the nightly and wait for 1.5.1

**Comment by patman13** [comment]
@bsongis-frsky Nightly works fine for me. Could you put this change into 1.5.3?

**Comment by bsongis-frsky** [comment]
Available in nightlies, and ready for 1.5.3
